stdlib/Dates/src/accessors.jl

```diff
@@ -41,6 +43,47 @@ function week(days)
     return div(w, 28) + 1
 end
 
+"""
+Year defined as ISO year. The equivalent is
+`week(lastdayofyear(dt) - Day(3))`  as 28 december is always in the last week
```

Comment:
This should probably go as a comment inside of the function. The docstring should be user-oriented, for example:

```suggestion
Return the number of ISO weeks in the given year (see https://en.wikipedia.org/wiki/ISO_week_date).
```